packages/react-devtools-shared/src/devtools/views/ErrorBoundary.js

```diff
@@ -152,3 +243,88 @@ export default class ErrorBoundary extends Component<Props, State> {
     this.setState(InitialState);
   };
 }
+
+function UpdateExistingIssue({gitHubItem}: {|gitHubItem: GitHubItem|}) {
+  return (
+    <div className={styles.GitHubLinkRow}>
+      <Icon className={styles.ReportIcon} type="bug" />
+      <div className={styles.UpdateExistingIssuePrompt}>
+        Update existing issue:
+      </div>
+      <a
+        className={styles.ReportLink}
+        href={gitHubItem.html_url}
+        rel="noopener noreferrer"
+        target="_blank"
+        title="Report bug">
+        {gitHubItem.title}
+      </a>
+    </div>
+  );
+}
+
+function ReportNewIssue({
+  callStack,
+  componentStack,
+  errorMessage,
+}: {|
+  callStack: string | null,
+  componentStack: string | null,
+  errorMessage: string | null,
+|}) {
+  let bugURL = process.env.GITHUB_URL;
+  if (!bugURL) {
+    return null;
+  }
+
+  const title = `Error: "${errorMessage || ''}"`;
+  const label = 'Component: Developer Tools';
+
+  let body = 'Describe what you were doing when the bug occurred:';
+  body += '\n1. ';
+  body += '\n2. ';
+  body += '\n3. ';
+  body += '\n\n---------------------------------------------';
+  body += '\nPlease do not remove the text below this line';
+  body += '\n---------------------------------------------';
+  body += `\n\nDevTools version: ${process.env.DEVTOOLS_VERSION || ''}`;
+  if (callStack) {
+    body += `\n\nCall stack: ${callStack.trim()}`;
+  }
+  if (componentStack) {
+    body += `\n\nComponent stack: ${componentStack.trim()}`;
+  }
+
+  bugURL += `/issues/new?labels=${encodeURI(label)}&title=${encodeURI(
+    title,
+  )}&body=${encodeURI(body)}`;
+
+  return (
+    <div className={styles.GitHubLinkRow}>
+      <Icon className={styles.ReportIcon} type="bug" />
+      <a
+        className={styles.ReportLink}
+        href={bugURL}
+        rel="noopener noreferrer"
+        target="_blank"
+        title="Report bug">
+        Report this issue
+      </a>
+      <div className={styles.ReproSteps}>
+        (Please include steps on how to reproduce it.)
```

Comment:
We may want to specify that this should include the app. What I often see is "open devtools, profile app". Which doesn't really help.

Reply:
Good point, although there's not a lot of space here to put a big prompt with instructions. Totally open to wording suggestions though!

I wish I could add a required field for e.g. Code Sandbox repro or something.

I guess I _could_ if I programmatically created the GH issue but then I'd need to stand up a backend service with an auth token and that's more involved than I want to do (also another point of failure).

Reply:
```suggestion
        (Please include steps on how to reproduce it and the components used.)
```

That's just a rough idea. Ideally we'd have some precedence what we call the app/UI/components.

Reply:
Sure. Let's try that and see.